src/pykeen/triples/triples_factory.py

```diff
@@ -764,12 +764,7 @@ def from_path(
         :return:
             A new triples factory.
         """
-        if isinstance(path, str):
-            path = os.path.abspath(path)
-        elif isinstance(path, TextIO):
```

Comment:
I don't remember why we were handling TextIO here

Reply:
Or anywhere

Reply:
I guess it was for being able to put in an opened file? Maybe it was just forwarded from the `numpy.loadtxt` interface :sweat_smile:

Reply:
Yeah I think it was just because numpy.loadtxt could handle it. That doesn't make so much sense everywhere though so It's time to start removing it